README.md

```diff
@@ -127,7 +127,7 @@ export GOPATH="$HOME/go"
 ### Intel® SGX SDK ###
 
 The Intel® SGX enclave implementation has been tested with Intel® SGX
-SDK for Linux version 2.1.1. For installation instuctions please visit
+SDK for Linux version 2.3.1. For installation instuctions please visit
 [installation guide][sgx-install]. A conventional directory to install
```

Comment:
Maybe also update the link to the installation guide. The link became broken. It seems to change quite often. Maybe this link would be more stable: https://01.org/intel-software-guard-extensions/downloads. Then it could be referred to as "download page" instead of "installation guide".